includes/class-wc-payments-payment-request.php

```diff
@@ -760,7 +762,7 @@ public function ajax_get_shipping_options() {
 			]
 		);
 		$product_view_options      = filter_input_array( INPUT_POST, [ 'is_product_page' => FILTER_SANITIZE_STRING ] );
-		$should_show_itemized_view = ! isset( $product_view_options['is_product_page'] ) ?: filter_var( $product_view_options['is_product_page'], FILTER_VALIDATE_BOOLEAN );
+		$should_show_itemized_view = ! isset( $product_view_options['is_product_page'] ) ? true : filter_var( $product_view_options['is_product_page'], FILTER_VALIDATE_BOOLEAN );
```

Comment:
This seems fine but could `||` be a replacement for `?:` here, since these are booleans?

Reply:
Only reason I changed it was because of phpcs:

> Using short ternaries is not allowed

Reply:
👍 I figured – just wondering if `||` could replace it in this case. Looks fine as it is though

Reply:
Oh, I see what you mean now. I misinterpreted your comment. I believe it could replace `?:`, yes.

> Looks fine as it is though

Ok 👍